File: socialnetworkingapp-front/src/app/account/network-entity.ts

export class NetworkEntity {

  firstName: string;
  lastName: string;
  email: string;
  imageUrl: string;
  position: string;
  company: string;

  constructor(firstName: string, lastName: string, email: string, imageUrl: string, position: string, company: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
    this.imageUrl = imageUrl;
    this.position = position;
    this.company = company;
  }
}

File: src/main/java/com/example/socialnetworkingapp/model/account/NetworkEntity.java
package com.example.socialnetworkingapp.model.account;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

@AllArgsConstructor
@Getter
@Setter
public class NetworkEntity implements Serializable {

    private String firstName;
    private String lastName;
    private String email;
    private String imageUrl;
}